src/lib/runAlloy.js

```diff
@@ -6033,7 +6033,7 @@ governing permissions and limitations under the License.
 
 // The value will be swapped with the proper version at build time
 // see rollupPluginReplaceVersion.js
-      var libraryVersion = "1.0.0";
```

Comment:
Nothing else changed?

Reply:
yes, comparing with master branch. I have updated the runAlloy.js file in my last PR related to remove syncIdentity command.

Reply:
Here I just bumped the versions

Reply:
Okay, cool. Looks good then.